nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,127 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the Install drive {#sec-installation-booting}
 
-NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-installation is by and large the same as a BIOS installation. The
-differences are mentioned in the steps that follow.
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Assuming your computer is still running, plug in the install drive
+and restart your computer.
```

Comment:
So from power-off I don't need to plug in the install drive?

(Not sure how to rephrase this, but it seems like there's a disconnect between the sentences.)